### ISSUE TITLE
Request for Training Code of Backbone

### ISSUE THREAD

**Issue by Chuchad** [Ekko-zn/Diff-based-AIGD#1]
Congratulations on the acceptance of your paper!  The logits used in the method are extracted from the backbone, which I believe is an important detail for reproducing the results. However, I couldn’t find the training code for the backbone, could you please consider making that part of the code public as well?

**Comment by Ekko-zn** [comment]
The training of the backbone involves a large dataset (real images and reconstructed images with multiple noise versions). In the future, we will consider making this dataset and the training code available for open source.

**Comment by Liy-s** [comment]
Hello, I have the same question as you, and I want to know if you have tried to reproduce the training code of the backbone by yourself or not. Thanks.

**Comment by zdaiot** [comment]
Same question~
